project/ScalafixBuild.scala

```diff
@@ -179,6 +180,11 @@ object ScalafixBuild extends AutoPlugin with GhpagesKeys {
   private val PreviousScalaVersion: Map[String, String] = Map(
   )
 
+  override def buildSettings: Seq[Setting[_]] = List(
+    versionScheme := Some("pvp"),
+    versionPolicyIntention := Compatibility.None // TODO: harden after 0.10.0
```

Comment:
I verified that with `Compatibility.BinaryCompatible`, we get
```
[error] Incompatibilities with dependencies of scalafix-core:0.9.34
[error]   com.geirsson:metaconfig-core_2.13: incompatible version change from 0.9.15 to 0.10.0 (compatibility: package versioning policy)
[error]   com.geirsson:metaconfig-typesafe-config_2.13: incompatible version change from 0.9.15 to 0.10.0 (compatibility: package versioning policy)
[error]   com.lihaoyi:fansi_2.13: incompatible version change from 0.2.14 to 0.3.0 (compatibility: package versioning policy)
[error]   com.lihaoyi:pprint_2.13: missing dependency
[error]   org.scalameta:common_2.13: incompatible version change from 4.4.32 to 4.5.0 (compatibility: package versioning policy)
[error]   org.scalameta:parsers_2.13: incompatible version change from 4.4.32 to 4.5.0 (compatibility: package versioning policy)
[error]   org.scalameta:scalameta_2.13: incompatible version change from 4.4.32 to 4.5.0 (compatibility: package versioning policy)
[error]   org.scalameta:trees_2.13: incompatible version change from 4.4.32 to 4.5.0 (compatibility: package versioning policy)
[error] Dependencies of module scalafix-core:0.9.34+56-5a4c18d3+20220319-2243-SNAPSHOT break the intended compatibility guarantees 'Compatibility.BinaryCompatible' (see messages above). You have to relax your compatibility intention by changing the value of versionPolicyIntention.
```

Reply:
opened https://github.com/scalacenter/scalafix/pull/1568/commits/79259b475cf4278a0440b9a2ebaa5b5e3bc0e3c6 as draft for a reminder

Reply:
You probably also want to add:

~~~ scala
versionPolicyIgnoredInternalDependencyVersions := Some("^\\d+\\.\\d+\\.\\d+\\+\\d+".r)
~~~

See https://github.com/scalacenter/sbt-version-policy#how-to-integrate-with-sbt-dynver